### PR TITLE
Put UCSDT2-A in indefinite downtime

### PR DESCRIPTION
**Pull request by jdost321**

### CODE REVIEW

**Comment by jdost321** [comment]
@brianhlin we will reinstall it eventually, Terrence just can't commit to a date yet so we put an artificially high year. Is that ok?

**Comment by brianhlin** [comment]
That's fine, whenever you reinstall it you can set `Active: True`

**Comment by brianhlin** [comment]
(note that we have this documented here: https://opensciencegrid.org/docs/common/registration/#registering-resource-downtimes)

File: topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml [hunk 1]
```diff
@@ -620,3 +620,4 @@
   - CE
 # ---------------------------------------------------------
 
+
```
Comment (brianhlin) [inline topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml]: Looks like there's still an extra whitespace change that snuck in

```suggestion
```

**Comment by brianhlin** [comment]
LGTM!

**Pull request merged**